pkgs/applications/video/kodi/unwrapped.nix

```diff
@@ -120,13 +92,19 @@ in stdenv.mkDerivation {
     pname = "kodi";
     version = kodiVersion;
 
-    src = kodi_src;
+    src = fetchFromGitHub {
+      owner = "xbmc";
+      repo = "xbmc";
+      rev = "${kodiVersion}-${rel}";
+      hash = "sha256-xrFWqgwTkurEwt3/+/e4SCM6Uk9nxuW62SrCFWWqZO0=";
+    };
+
     buildInputs = [
       gnutls libidn2 libtasn1 nasm p11-kit
       libxml2 python3Packages.python
       boost libmicrohttpd
       gettext pcre-cpp yajl fribidi libva libdrm
-      openssl gperf tinyxml2 taglib libssh
+      openssl gperf tinyxml2 tinyxml-2 taglib libssh
```

Comment:
Do we add both `tinyxml2` and `tinyxml-2` on purpose?

Reply:
unfortunately it doesn't build without
i assume they started migrating and haven't finished yet... but that's just an assumption - it might be worth reading the code a bit to validate that

let's add that as a to-do item which can happen either before or after this PR is merged
i just want to be sensitive to nixos freeze coming up soon

Reply:
yes, both still needed:

https://github.com/xbmc/xbmc/blob/4d70847511fc1461a12218020b57c77b861c1a67/CMakeLists.txt#L223-L224